src/main/java/com/minecolonies/api/colony/ICitizenDataView.java

```diff
@@ -220,4 +220,12 @@ public interface ICitizenDataView extends ICitizen
      * @param position the pos of the home building.
      */
     void setHomeBuilding(BlockPos position);
+
+    /**
+     * Get the colony this citizen belongs to.
+     *
+     * @return the colony view.
+     */
+    @Override
+    IColonyView getColony();
```

Comment:
A bit weird. Why is the existing IColony handling one not good enough?

Reply:
This one ensures that the `getColony()` always returns the IColonyView so you don't have to unnecessarily cast.